crates/lib/src/db/def.rs

```diff
@@ -11,7 +11,12 @@ use spacetimedb_sats::{de, ser};
 use std::sync::Arc;
 
 /// The default preallocation amount for sequences.
-pub const SEQUENCE_PREALLOCATION_AMOUNT: i128 = 4_096;
+///
+/// Start with no values allocated. The first time we advance the sequence,
+/// we will allocate [`SEQUENCE_ALLOCATION_STEP`] values.
+pub const SEQUENCE_PREALLOCATION_AMOUNT: i128 = 0;
```

Comment:
This looks unused, if I'm correct. Could you please remove it.

Reply:
Ah it's used in this file. Could you please rename to `SEQUENCE_INITIAL_PREALLOCATION_AMOUNT`?

Reply:
I eliminated this constant, under the theory that writing `allocation: 0` at its usage is more explicit.